<launch>
    <node pkg="homework" type="sensor_node.py" name="sensor_node" output="screen"></node>
    <node pkg="homework" type="mission_node.py" name="mission_node"></node>
    <node pkg="homework" type="controller.py" name="controller_node" output="screen">
    <param name="linear_speed" value=".8" />
    </node>

    <node pkg="homework" type="monitor.py" name="monitor"></node>


    <include file="$(find homework)/launch/turtlebot3_world.launch">
        <arg name="x_pos" value="0"/>
        <arg name="y_pos" value="0"/>
        <arg name="z_pos" value="0.0"/>
    </include>

    <include file="$(find turtlebot_rviz_launchers)/launch/view_robot.launch"/>

   
</launch>
